tests/queries/0_stateless/02141_clickhouse_local_interactive_table.reference

```diff
@@ -1,2 +1,2 @@
-CREATE TABLE default.`table`\n(\n    `key` String\n)\nENGINE = File(\'TSVWithNamesAndTypes\', \'/dev/null\')
-CREATE TABLE foo.`table`\n(\n    `key` String\n)\nENGINE = File(\'TSVWithNamesAndTypes\', \'/dev/null\')
```

Comment:
What was wrong with non-temporary table here?

Reply:
Now clickhouse-local uses temporary tables for inputs.